lib/memcached/memcached.rb

```diff
@@ -453,6 +460,9 @@ def cas(keys, ttl=@default_ttl, decode=true)
     else
       # Single CAS
       value, flags, cas = single_get(keys, decode)
+      if !@raise_on_errors && value.equal?(NOT_FOUND)
```

Comment:
```suggestion
      if !@raise_on_errors && NOT_FOUND == value
```

micro-optimization: you'd think `.equal?` would be the fastest thing ever, but actually it is slower than `==` in many cases because it doesn't have a dedicated `opt_` opcode like `==` does.